gruntwork-install

```diff
@@ -230,12 +232,12 @@ function run_module {
   local readonly module_name="$1"
   local readonly download_dir="$2"
   shift 2
-  local readonly module_params=($@)
+  local -ra module_params=("$@")
```

Comment:
Ugh, good catch

Reply:
Thanks, I went back fixed all of these and then rebased onto that commit. See #46